Backend/app/therapy_sessions/serializers.py

```
@@ -0,0 +1,738 @@
+from rest_framework import serializers
+from django.contrib.auth import get_user_model
+from .models import (
+    Session, SessionTemplate, PatientProgress, SessionReminder,
+    TherapistAvailability, SessionQRCode, SessionAudio, SessionInsight
```

Ruff (0.12.2)

`.models.SessionQRCode` imported but unused

Remove unused import

(F401)